docs/developers-clear-containers-install.md

```diff
@@ -0,0 +1,141 @@
+# Developers Clear Containers 3.0 Install
+
+This guide is not intended for end-users. Instead, this guide provides
+instructions for any developers eager to try Clear Containers 3.0 and who
+want to build Clear Containers from the source code and are familiar with the
+process. Basic programming knowledge and familiarity with the Linux command
+line are expected.
+
+## Requirements
+
+  * [go 1.8.3](https://golang.org/)
+  * [glibc-static](https://www.gnu.org/software/libc/libc.html) Required to build **Virtcontainers pause**
```

Comment:
removed in the new commit I think is clearer, and there is an explanation why we wil install only pause from Virtcontainers project